ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -142,6 +142,6 @@
     private CCDLaList assignedTo;
     private CCDContactPartyType contactChangeParty;
     private CCDContactChangeContent contactChangeContent;
-    private CCDProceedOnPaperRequestType proceedOnPaperReason;
+    private CCDProceedOnPaperReasonType proceedOnPaperReason;
```

Comment:
Property name should be 'proceedOnPaperReasonType'

Reply:
I feel not necessary. Type says it is a type (Enum). If you see the other properties too, PartyType is the property type and changeParty is the property name